Modules/Relationships/src/Relationships.Application/Relationships/DTOs/RelationshipMetadataDTO.cs

```diff
@@ -25,6 +26,7 @@ public RelationshipMetadataDTO(Relationship relationship)
 
     public DateTime CreatedAt { get; set; }
 
+    public byte[]? CreationResponseContent { get; set; }
```

Comment:
Why did you add this? The content hasn't been there before intentionally. This is the Relationship**Metadata**DTO, which means it doesn't have content. It is used for responses where we don't care about the content, namely the responses of all actions.

Reply:
My bad, I misunderstood the requirements then.

Changes made.